src/server/pegasus_write_service_impl.h

```diff
@@ -182,59 +182,61 @@ class pegasus_write_service::impl : public dsn::replication::replica_base
         uint32_t new_expire_ts = 0;
         db_get_context get_ctx;
         int err = db_get(raw_key, &get_ctx);
-        if (err == 0) {
-            if (!get_ctx.found) {
-                // old value is not found, set to 0 before increment
-                new_value = update.increment;
-                new_expire_ts = update.expire_ts_seconds > 0 ? update.expire_ts_seconds : 0;
-            } else if (get_ctx.expired) {
-                // ttl timeout, set to 0 before increment
-                _pfc_recent_expire_count->increment();
+        if (err != 0) {
+            resp.error = err;
```

Comment:
The line seem to be useless?

Reply:
Certainly useful. It sets response error.

Reply:
But you `return error` , `resp.error` is not be used.

Reply:
Well, I konw